assets/js/app.js

```diff
@@ -49,12 +49,21 @@ Hooks.Items = {
     })
 
     this.el.addEventListener("update-indexes", e => {
-        console.log('yyyyy')
         const ids = [...document.querySelectorAll(".item")].map( i => i.dataset.id)
         hook.pushEventTo("#items", "updateIndexes", {ids: ids})
     })
   }
 }
+
+Hooks.Counter = {
+  mounted() {
+    const hook = this
```

Comment:
Unclear why you're defining `hook = this` when you're then attaching an event listener to `this` ... 🤷‍♂️

Reply:
My `js` is a bit rusty, I took this structure from the documentation (I'll try to find exactly where again) but if I'm not mistaken is to be able to use `hook.pushEventTo` below. If I use `this.pushEvent` then `this` will reference the event listened to instead of the hook.

Reply:
LoL! Ironically, if your `JS` was `Rust`-y you wouldn't be creating a local copy of the global object. 😜 

Just tested `this` without the `const hook` on my `localhost` and it works fine in all browsers without any issues. 👌 

PR: https://github.com/dwyl/learn-alpine.js/pull/19 ✂️